scripts/init.sh

```diff
@@ -130,104 +83,147 @@ function log_success_msg() {
 function start() {
     # Check if config file exist
     if [ ! -r $CONFIG ]; then
-        log_failure_msg "config file doesn't exist (or you don't have permission to view)"
+        log_failure_msg "config file $CONFIG doesn't exist (or you don't have permission to view)"
         exit 4
     fi
 
-    # Checked the PID file exists and check the actual status of process
-    if [ -e $PIDFILE ]; then
-	PID="$(pgrep -f $PIDFILE)"
-	if test -n "$PID" && kill -0 "$PID" &>/dev/null; then
-	    # If the status is SUCCESS then don't need to start again.
-            log_failure_msg "$NAME process is running"
-            exit 0 # Exit
+    # Check that the PID file exists, and check the actual status of process
+    if [ -f $PIDFILE ]; then
+	      PID="$(cat $PIDFILE)"
+	      if kill -0 "$PID" &>/dev/null; then
+	          # Process is already up
+            log_success_msg "$NAME process is already running"
```

Comment:
All weird indentation should be fixed. Looks like my editor mixed in tabs and spaces for indentation.